pkg/logs/input/file/tailer_windows.go

```diff
@@ -95,3 +96,22 @@ func (t *Tailer) readForever() {
 		}
 	}
 }
+
+func openFileShareDeleteMode(path string) (*os.File, error) {
+	pathp, err := syscall.UTF16PtrFromString(path)
```

Comment:
Could we please add a detailed comment explaining the issue?

Reply:
👍